src/workerd/api/worker-loader-test.js

```diff
@@ -32,6 +32,41 @@ export let basics = {
   },
 };
 
+export let pythonBasics = {
+  async test(ctrl, env, ctx) {
+    let worker = env.loader.get('pythonBasics', () => {
+      return {
+        compatibilityDate: '2025-01-01',
+        mainModule: 'foo.py',
+        compatibilityFlags: ['python_workers'],
+        modules: {
+          'foo.py': `
+from workers import WorkerEntrypoint
+# TODO: workerd bug with default entrypoint
```

Comment:
This will be fixed by https://github.com/cloudflare/workerd/pull/4616

Reply:
Looks like that was merged, can this be updated?